core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -185,6 +195,9 @@ public void configure() throws ConfigurationException {
 						pipe.registerForward(pf);
 					} else {
 						PipeLineExit plexit = findExitByState(ExitState.SUCCESS);
+						if (plexit != null) {
+							plexit = pipeLineExits.values().iterator().next(); // if there is no success exit, then just get the first one
+						}
```

Comment:
Deze snap ik niet helemaal? als er niets is gedefineerd, wat is dan de eerste exit?